addon-test-support/rows-fetcher.ts

```diff
@@ -0,0 +1,26 @@
+export default class RowsFetcher {
+  // @ts-ignore
```

Comment:
Do you keep this?

Reply:
Yep! it's because the args are not used, but i wanted to keep them even if t's a mock